include/odp/api/abi-default/packet.h

```diff
@@ -34,6 +34,7 @@ typedef _odp_abi_packet_vector_t *odp_packet_vector_t;
 #define ODP_PACKET_SEG_INVALID    ((odp_packet_seg_t)0)
 #define ODP_PACKET_OFFSET_INVALID 0xffff
 #define ODP_PACKET_VECTOR_INVALID   ((odp_packet_vector_t)0)
+#define ODP_PACKET_MAX_FLOW_HASH  UINT32_MAX
```

Comment:
max flow hash value could be a pktio capability. pktio max hash >= flow aware scheduler max flow hash capa. This would be also the max flow hash value that application can set into packet.

Reply:
Thanks for reviewing, I will get back with V2.